client/src/client-only-routes/ShowProjectLinks.js

```diff
@@ -0,0 +1,196 @@
+          })
+        )
+      })
+    ),
+    username: PropTypes.string
+  }),
+  username: PropTypes.string
+};
+
+const initSolutionState = {
+  projectTitle: '',
+  files: null,
+  solution: null,
+  isOpen: false
+};
+
+const ShowProjectLinks = props => {
+  const [solutionState, setSolutionState] = useState(initSolutionState);
+
+  const handleSolutionModalHide = () => setSolutionState(initSolutionState);
+
+  const getProjectSolution = (projectId, projectTitle) => {
+    const {
+      user: { completedChallenges }
+    } = props;
+
+    const completedProject = find(
+      completedChallenges,
+      ({ id }) => projectId === id
+    );
+
+    if (!completedProject) {
+      return null;
+    }
+
+    const { solution, githubLink, files } = completedProject;
+    const onClickHandler = () =>
+      setSolutionState({
+        projectTitle,
+        files,
+        solution,
+        isOpen: true
+      });
+
+    if (files && files.length) {
+      return (
+        <button
+          onClick={onClickHandler}
+          className='project-link-button-override'
+        >
+          solution
+        </button>
+      );
+    }
+    if (githubLink) {
+      return (
+        <>
+          <a href={solution} rel='noopener noreferrer' target='_blank'>
+            solution
+          </a>
+          ,{' '}
+          <a href={githubLink} rel='noopener noreferrer' target='_blank'>
+            source
+          </a>
+        </>
+      );
+    }
+    if (maybeUrlRE.test(solution)) {
+      return (
+        <a
+          block={'true'}
+          className='btn-invert'
+          href={solution}
+          rel='noopener noreferrer'
+          target='_blank'
+        >
+          solution
+        </a>
+      );
+    }
+    return (
+      <button className='project-link-button-override' onClick={onClickHandler}>
+        solution
+      </button>
+    );
+  };
+
+  const renderCertification = certName => (
+    <ul key={certName}>{renderProjectsFor(certName)}</ul>
+  );
+
+  const renderProjectsFor = certName => {
+    if (certName === 'Legacy Full Stack') {
+      const legacyCerts = [
+        { title: 'Responsive Web Design' },
+        { title: 'JavaScript Algorithms and Data Structures' },
+        { title: 'Front End Libraries' },
+        { title: 'Data Visualization' },
+        { title: 'APIs and Microservices' },
+        { title: 'Legacy Information Security and Quality Assurance' }
+      ];
+      return legacyCerts.map((cert, ind) => {
+        const mapToUse = projectMap[cert.title] || legacyProjectMap[cert.title];
+        const { superBlock } = first(mapToUse);
+        const certLocation = `/certification/${username}/${superBlock}`;
+        return (
+          <li key={ind}>
+            <a
+              href={certLocation}
+              className='btn-invert'
+              rel='noopener noreferrer'
+              target='_blank'
+            >
+              {cert.title}
+            </a>
```

Comment:
```suggestion
            <Link to={certLocation}>{cert.title}</Link>
```
Should be fine if we use the helper.